docs/get_started/plan_deployment/traditional_deployment/plan_multiple_profile.md

```diff
@@ -1,47 +1,47 @@
-# Multiple profile support
+# Multiple-profile support
 
-Multiple profiles give you the ability to have multiple, independently configured portal instances that run from the same installation.
+With multiple profiles you can work with multiple, independently configured portal instances that run on the same installation.
 
-Starting with Portal Version 7, it is possible to create extra profiles in addition to the default profile, which is useful in the following scenarios:
+Starting with Portal Version 7, you can create extra profiles in addition to the default profile. Extra proviles are useful in the following scenarios:
 
--   Reuse a single installation for multiple independent portal instances, such as for various test or development efforts.
--   Recover from a configuration problem by deleting the current profile and re-creating it without having to reinstall the product.
+-   Reuse a single installation for multiple, independent portal instances, such as for various test or development efforts.
+-   Recover from a configuration problem by deleting the current profile and re-creating it without reinstalling the product.
 -   Update a Deployment Manager profile to handle portal servers and thus avoid all the manual preparation steps.
 
-The following profile types can be generated:
+You can generate the following profile types:
 
 -   **portal.default**
 
-    This profile holds a stand-alone portal server in the captured configuration. It might be used as a stand-alone server or after the federation as a base for a portal cluster.
+    This profile holds a stand-alone portal server in the captured configuration. The server might be used as a stand-alone server or, after federation, as a base for a portal cluster.
```

Comment:
Can you please remove the coma "server or, after federation, ".

Reply:
No need to remove comma